src/main/java/io/github/vcuswimlab/stackintheflow/model/L2H/CustomL2H.java

```diff
@@ -0,0 +1,2231 @@
+package io.github.vcuswimlab.stackintheflow.model.L2H;
```

Comment:
Does this file need to be a part of the plugin itself? (i.e. is it necessary for it to be bundled in the jar that we ship out to users?)  If it is only for use in a per-processing step to generate the model and it not utilized afterwards it should **not** be included in the main project repo.  We should instead create a sub-project for it.

Reply:
Yes, L2HPredictor uses the CustomL2H class to make it's predictions. It is not a pre-processing step. No pre-processing code is included in the pull request.

Reply:
That's true, the plugin doesn't need that particular part of the CustomL2H class. I've removed the main method and everything that was only used by the main method. It wasn't a drastic reduction, but less code is still less code.